packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Models/Friend.Serialization.cs

```diff
@@ -24,25 +26,127 @@ internal Friend()
         {
         }
 
-        void IJsonModel<Friend>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
+        void IJsonModel<Friend>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options) => WriteCore(writer, options);
+
+        Friend IJsonModel<Friend>.Create(ref Utf8JsonReader reader, ModelReaderWriterOptions options) => CreateCore(ref reader, options);
+
+        BinaryData IPersistableModel<Friend>.Write(ModelReaderWriterOptions options) => WriteCore(options);
```

Comment:
I think we want to name these to match their interfaces since we have multiple Write methods we are explicitly implementing.

Lets go with PersistableModelWriteCore and JsonModelWriteCore.